assets/admin/emails/email-preview-button/email-preview-button.test.js

```diff
@@ -0,0 +1,115 @@
+/**
+ * External dependencies
+ */
+import { fireEvent, render, screen, act } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import { EmailPreviewButton } from './email-preview-button';
+
+const defaultProps = {
+	previewLink: 'https://example.com/',
+	postId: 1,
+	isSaveable: true,
+	isAutosaveable: true,
+	isLocked: false,
+	isDraft: false,
+	autosave: jest.fn(),
+	savePost: jest.fn(),
+};
+
+describe( '<EmailPreviewButton />', () => {
+	const { getByText, queryByText } = screen;
+
+	beforeEach( () => {
+		document.body.innerHTML = `<div class="block-editor-post-preview__dropdown" />`;
+	} );
+
+	it( 'Should display the preview button', async () => {
+		render( <EmailPreviewButton { ...defaultProps } /> );
+
+		expect( getByText( 'Preview' ) ).toBeInTheDocument();
+	} );
+
+	it( "Shouldn't display the preview button when the container is missing", async () => {
+		document.body.innerHTML = '';
+
+		render( <EmailPreviewButton { ...defaultProps } /> );
+
+		expect( queryByText( 'Preview' ) ).toBeNull();
+	} );
+
+	it( 'Should open a new window when the button is clicked', async () => {
+		global.open = jest.fn();
+
+		render( <EmailPreviewButton { ...defaultProps } /> );
+
+		await act( async () => {
+			fireEvent.click( getByText( 'Preview' ) );
```

Comment:
I think you can use "userEvent.click"  so the act calls are required. 
`import userEvent from '@testing-library/user-event';`

Reply:
Updated in https://github.com/Automattic/sensei/pull/6583/commits/5e3b51a60f382c4da34f60cfd4dc87f97b40e65a. Thanks!